src/traces/parcoords/devtesting/colors.js

```diff
@@ -0,0 +1,36 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+'use strict';
+var jet = [[0, 0, 255], [0, 8, 255], [0, 35, 255], [0, 77, 255], [0, 126, 255], [0, 175, 255], [0, 217, 255], [0, 245, 255], [0, 254, 255], [0, 255, 245], [0, 255, 218], [0, 255, 176], [0, 255, 128], [0, 255, 78], [0, 255, 36], [0, 255, 8], [0, 255, 0], [8, 255, 0], [36, 255, 0], [77, 255, 0], [127, 255, 0], [176, 255, 0], [217, 255, 0], [245, 255, 0], [254, 255, 0], [255, 245, 0], [255, 217, 0], [255, 176, 0], [255, 127, 0], [255, 77, 0], [255, 36, 0], [255, 8, 0], [255, 0, 0]
```

Comment:
This is in the `devtesting` directory. The entire directory will go. It's not being used (e.g. our example uses `"Jet"` or whichever `colorscale` you want. This was one of your main feedback items. The reason for `devtesting` is that it avoided me spending 10-15 seconds on each rebuild, letting me get a refreshed view in 1-2 seconds. I'm locally using `budo` for this.